FAQ/FAQ-LightningNetwork.md

```diff
@@ -318,6 +318,58 @@ cd btcpayserver-docker
 export LIGHTNING_ALIAS="namehere"
 . ./btcpay-setup.sh -i
 ```
+### How to back up LND channels offsite
+Currently your BTCPay Server docker on every change in channels state saves static channels backup (SCB) locally on the same disk (on-site). This poses a threat to LN off-chain funds recovery in case of the server crash or fatal data loss.
```

Comment:
Capital `Docker` and would be good to include a link to it https://github.com/btcpayserver/btcpayserver-docker